style.css

```diff
@@ -360,6 +360,24 @@ h4 {
   color: gray;
 }
 
+.modal1 {
+  background-color: #fff;
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  visibility: hidden;
+}
+
+.modal1.openbtn1 {
+  visibility: visible;
+}
```

Comment:
- [ ] Please make sure that your design matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. (Also the mobile version)


### Your design
![image](https://user-images.githubusercontent.com/83803414/196972412-295000c5-1e45-4f10-8698-10596bd9448f.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/196972609-96738090-ffcf-41fd-a4b6-f246f7599da3.png)